packages/core/src/attestation/Attestation.ts

```diff
@@ -138,7 +138,8 @@ export function verifyAgainstCredential(
 ): void {
   if (
     credential.claim.cTypeHash !== attestation.cTypeHash ||
-    credential.rootHash !== attestation.claimHash
+    credential.rootHash !== attestation.claimHash ||
```

Comment:
I think it would be nice to report which entry didn't match. A dev using the SDK and debugging a potential issue might appreciate that information.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/702/commits/818f32c03eb0445a0b8e37efa7bcd12d80148694